### PR TITLE
ECOM-7021 Convert organizations tags to typeahead and add tags to Course Runs

### PR DESCRIPTION
**Pull request by MatthewPiatetsky**
@edx/ecommerce 

tags4

### CODE REVIEW

**Review by rlucioni — APPROVED** [review]
@MatthewPiatetsky very nice.

@mikedikan this is a good example of an area where some pre-planning/technical discovery would have done us good. I think we spent an hour talking about this yesterday?

File: requirements/base.txt [hunk 1]
```diff
@@ -18,6 +18,7 @@ django-sortedm2m==1.3.2
 django-stdimage==2.3.3
 django-storages==1.5.0
 django-taggit==0.20.2
```
Comment (clintonb) [inline requirements/base.txt]: Let's take this opportunity to update this library: https://pypi.python.org/pypi/django-taggit.

**Review by clintonb — APPROVED** [review]

**Comment by MatthewPiatetsky** [comment]
@rlucioni to be fair, i think we spent most of the time talking about the approach and figuring out that tags is what we wanted, and only a few minutes about whether tags was actually possible. Regardless, in the future I think if we decide we want a certain approach but aren't sure how long it could take we could just write "try to do x and come back to the team to discuss if it seems like it will be very complex"

File: requirements/base.txt (hunk 1 above)
Comment (MatthewPiatetsky) [reply requirements/base.txt]: updated

**Comment by rlucioni** [comment]
@MatthewPiatetsky fair, that's a good point.

**Comment by mikedikan** [comment]
@rlucioni I have received your feedback and appreciate your giving of it.

**Comment by rlucioni** [comment]
@mikedikan 👍

**Pull request merged**